platform-sdk/swirlds-base/src/main/java/com/swirlds/base/time/TimeFacade.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.base.time;
+
+import com.swirlds.base.time.internal.NanoClock;
+import com.swirlds.base.time.internal.OSTime;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.time.Clock;
+
+/**
+ * Facade for all time functionality that is part of the base module
+ */
+public final class TimeFacade {
```

Comment:
I think `Facade` is not the right name here.

Reply:
https://github.com/hashgraph/hedera-services/issues/5845

Reply:
well, it is a facade for the package. In general it is the "public entry point". If you have a better idea I'm fine :)

Reply:
Maybe `TimeFactory`?

When I hear `Facade` I think of an adapter that simplifies a complex API, offers a modern approach to an existing API or something similar. But here, we just have two factory methods.

Reply:
Why can't we just use the parent OSTime.getInstance() class? <- This class contains implementation details that I would like to hide.

Reply:
changed to `TimeFactory`